### PR TITLE
feat(kebbeh): change nprogress bar color

### PR DESCRIPTION
**Pull request by sohank1**

### CODE REVIEW

**Comment by sohank1** [comment]
I think color is too dark but i was told to use `color-primary-600` or `color-primary-700`

**Comment by BronzW** [comment]
I don't know about other people but I think it would be helpful if you created a new variable named something like progress-bar-color that uses --color-primary-600 and set all the current ones to that variable

**Comment by sohank1** [comment]
> I don't know about other people but I think it would be helpful if you created a new variable named something like progress-bar-color that uses --color-primary-600 and set all the current ones to that variable

What do you mean? Like things that use --color-primary-600 use this new var? I don't think that's a good idea

File: kibbeh/src/styles/globals.css [hunk 1]
```diff
@@ -116,6 +116,19 @@ textarea {
   z-index: 9999999;
 }
 
+#nprogress .bar {
+  background: var(--color-primary-600) !important;
```
Comment (amitojsingh366) [inline kibbeh/src/styles/globals.css]: why this color its barely even visible?

**Review by amitojsingh366 — CHANGES_REQUESTED** [review]

File: kibbeh/src/styles/globals.css (hunk 1 above)
Comment (sohank1) [reply kibbeh/src/styles/globals.css]: https://github.com/benawad/dogehouse/pull/2607#issuecomment-833746758  I was told to make it this color

Comment (sohank1) [reply kibbeh/src/styles/globals.css]: Changed it and I think it looks better now

**Comment by sohank1** [comment]
@amitojsingh366 Is it good now?
image

**Comment by amitojsingh366** [comment]
> @amitojsingh366 Is it good now?
> image

@ajmnz

**Comment by benawad** [comment]
I'll update the color later if ajb wants a diff one

**Pull request merged**